semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/messaging/MessagingSemanticConventionUtils.java

```diff
@@ -109,6 +111,28 @@ public static boolean isKafkaBackend(Event event) {
     return Lists.newArrayList(Sets.newHashSet(MESSAGING_OPERATION));
   }
 
+  public static List<String> getAttributeKeysForRabbitmqCommand() {
+    return Lists.newArrayList(Sets.newHashSet(RABBITMQ_COMMAND_VALUE));
+  }
+
+  public static String getMessagingOperation(Event event) {
+    return SpanAttributeUtils.getFirstAvailableStringAttribute(
+        event, MessagingSemanticConventionUtils.getAttributeKeysForMessagingOperation());
+  }
+
+  public static String getRabbitmqOperation(Event event) {
+    String messagingOperation =
```

Comment:
Can re-use the above method `getMessagingOperation` here.

Reply:
done